rfcs/0001-bootstrapping-rfc-process.md

```diff
@@ -0,0 +1,54 @@
+- Feature Name: bootstrapping-rfc-process
+- Start Date: 09 February 2018
+- RFC PR: (leave this empty)
+- IPFS Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Begin using an RFC process. Initially use a structure similar to the process used by the maintainers of IPFS lang, as described in https://github.com/IPFS-lang/rfcs
```

Comment:
This should be `https://github.com/ipfs/rfcs`, right? (not `IPFS-lang/rfcs`)

Reply:
Or is this actually supposed to be a reference to Rust, since you’re saying that’s what this is similar to? In either case, the link is wrong :P